src/applications/simple-forms/21-0966/tests/config/helpers.unit.spec.js

```diff
@@ -107,11 +114,25 @@ describe('form helper functions', () => {
   });
 });
 
+describe('initializeFormDataWithPreparerIdentification', () => {
+  it('returns an initialized formData object with preparerIdentification selection', () => {
+    expect(
+      initializeFormDataWithPreparerIdentification(
+        preparerIdentifications.veteran,
+      ),
+    ).to.deep.equal({
+      ...createInitialState(formConfig).data,
+      preparerIdentification: preparerIdentifications.veteran,
+    });
+  });
+});
+
 describe('confirmation page helper functions', () => {
   describe('Compensation claim type', () => {
     const data = {
-      benefitSelection: 'Compensation',
+      benefitSelection: {},
     };
+    data.benefitSelection[veteranBenefits.compensation] = true;
```

Comment:
i think you can also declare these directly in the object if you wanted to e.g.
benefitSelection: {
   [veteranBenefits.compensation]: true
}